Index: horizon-2015.1.1/openstack_dashboard/openstack/common/_i18n.py
===================================================================
--- horizon-2015.1.1.orig/openstack_dashboard/openstack/common/_i18n.py
+++ horizon-2015.1.1/openstack_dashboard/openstack/common/_i18n.py
@@ -10,13 +10,13 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-"""oslo.i18n integration module.
+"""oslo_i18n integration module.
 
-See http://docs.openstack.org/developer/oslo.i18n/usage.html
+See http://docs.openstack.org/developer/oslo_i18n/usage.html
 
 """
 
-import oslo.i18n
+import oslo_i18n
 
 
 # NOTE(dhellmann): This reference to o-s-l-o will be replaced by the
@@ -24,7 +24,7 @@ import oslo.i18n
 # repository. It is OK to have more than one translation function
 # using the same domain, since there will still only be one message
 # catalog.
-_translators = oslo.i18n.TranslatorFactory(domain='openstack_dashboard')
+_translators = oslo_i18n.TranslatorFactory(domain='openstack_dashboard')
 
 # The primary translation function using the well-known name "_"
 _ = _translators.primary
