Index: horizon-2015.1.1/openstack_dashboard/openstack/common/log.py
===================================================================
--- horizon-2015.1.1.orig/openstack_dashboard/openstack/common/log.py
+++ horizon-2015.1.1/openstack_dashboard/openstack/common/log.py
@@ -38,7 +38,7 @@ import sys
 import traceback
 
 from oslo_config import cfg
-from oslo.serialization import jsonutils
+from oslo_serialization import jsonutils
 from oslo_utils import importutils
 import six
 from six import moves
Index: horizon-2015.1.1/openstack_dashboard/openstack/common/policy.py
===================================================================
--- horizon-2015.1.1.orig/openstack_dashboard/openstack/common/policy.py
+++ horizon-2015.1.1/openstack_dashboard/openstack/common/policy.py
@@ -81,7 +81,7 @@ import os
 import re
 
 from oslo_config import cfg
-from oslo.serialization import jsonutils
+from oslo_serialization import jsonutils
 import six
 import six.moves.urllib.parse as urlparse
 import six.moves.urllib.request as urlrequest
